thrift/lib/cpp/transport/TFileTransport.cpp

```diff
@@ -523,7 +523,7 @@ void TFileTransport::writerThread() {
 
 void TFileTransport::flush() {
   // file must be open for writing for any flushing to take place
-  if (writerThreadId_ <= 0) {
+  if (writerThreadId_ == 0) {
```

Comment:
shouldn't this be `if (writerThreadId_ == pthread_zero)` instead or `if (!writerThreadId_)`?